client/src/RatingsAndReviews/Reviews/AddReview.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class AddReview extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      recommendThis: ''
+    }
+    this.createRadioOptions = this.createRadioOptions.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    const { target: { stateVal, value } } = event;
+    this.setState({
+      [stateVal]: value
+    });
+  }
+
+  createRadioOptions(arr, name, stateVal) {
```

Comment:
Wow, this is a cool way to go about creating all these radio buttons. I don't fully understand the syntax from line 15, but it is something I want to look into